Extension/package.json

```diff
@@ -2333,21 +2333,21 @@
           "markdownDescription": "%c_cpp.configuration.simplifyStructuredComments.markdownDescription%",
           "scope": "application"
         },
-        "C_Cpp.autocompleteDoxygenComment":{
-          "type":"boolean",
-          "default":true,
-          "description": "%c_cpp.configuration.autocompleteDoxygenComment.description%",
+        "C_Cpp.generateDoxygenCommentsOnTyping":{
```

Comment:
Should we do a category for doxygen settings?

e.g. `C_Cpp.doxygen.generateOnType`, `C_Cpp.doxygen.commentStyle`

Reply:
I like that idea. Makes it easier to find settings. We do that style for other recent features like codeAnalysis and vcFormat.

Reply:
but FYI, there is some spot in the settings update code that would need to change to handle the "." correctly

Reply:
...it may not matter now with 2 settings, but it could be more of a settings organization issue if we add more doxygen-related settings in the future.